conf/gulp-tasks/templates.gulpfile.js

```diff
@@ -106,15 +106,17 @@ async function createTemplatesForLanguages (languages, isSearchBarOnly = false)
  * Creates a template bundle for each supported language, locale pair.
  *
  * @param {boolean} isSearchBarOnly If only templates related to the SearchBar should be included.
+ * @param {string[]} languages a list of languages to build template bundle for,
+ *                             with the corresponding language-locale pairs.
  * @returns {Promise<Function>}
  */
-function allLocaleTemplates (isSearchBarOnly = false) {
+function allLocaleTemplates (isSearchBarOnly = false, languages = ALL_LANGUAGES) {
```

Comment:
is it necessary to define a default here if the functions that call this all do the same defaulting?

Reply:
will update to remove the default here